### ISSUE TITLE
TypeScript: Cannot find name 'Response'

### ISSUE THREAD

**Issue by Llorx** [d-fischer/twitch#102]
## Bug Report

When trying to compile something that uses `twitch` npm library, this problem raises.

### Code

After npm install:
```
import TwitchClient from "twitch";
let client = TwitchClient.withClientCredentials(config.twitch.id, config.twitch.secret);
```

### Expected behavior

Compile correctly.

### Actual Behavior

The error is thrown:
![image](https://user-images.githubusercontent.com/702490/74866338-033de380-5353-11ea-8d27-d2a7677abb8a.png)

### Environment

- Version: 4.0.2
- TypeScript: 3.7.5
- Node version: v12.16.1
- Operating system: Windows 10
- Only types installed are: "@types/node": "^12.12.27".

**Comment by d-fischer** [comment]
This is a known issue which I just added to the brand new FAQ page on the docs:
https://d-fischer.github.io/twitch/docs/basic-usage/faq.html

Leaving this open to track an actual fix.

**Comment by d-fischer** [comment]
Released in 4.1.0-pre.10.
